tools/buildPackage.js

```diff
@@ -19,11 +19,24 @@ var env = {
   NODE_ENV: 'production'
 }
 
+console.log('Cleaning build target')
+
+if (isWindows) {
+  var cmds = [
+    'if exist ' + buildDir + ' rmdir /s /q ' + buildDir,
+  ]
+  execute(cmds, env, console.log.bind(null, 'done'))
+} else {
+  var cmds = [
+    'rm -rf ' + buildDir,
+    'rm -f dist/*.dmg dist/*.nupkg dist/*.exe dist/*.msi dist/RELEASES dist/*.zip',
+  ]
+  execute(cmds, env, console.log.bind(null, 'done'))
+}
```

Comment:
I think i'll nix the cleaning message due to the async nature of execute. (I'm fairly new at node.js, is there a way to make execute serialize for things like cleanup that should finish before continuing?)